test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs

```diff
@@ -11,55 +11,93 @@
 using static Microsoft.NET.TestFramework.Commands.MSBuildTest;
 using System.Runtime.InteropServices;
 using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.NET.Build.Tasks;
+using FluentAssertions;
+
 namespace Microsoft.NET.Publish.Tests
 {
     public class GivenThatWeWantToCacheAProjectWithDependencies : SdkTest
     {
+        private static string libPrefix;
+        private static string runtimeos;
```

Comment:
All these field names should conform to our coding standards.  See [https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)